docs/user-guide/timeouts/timeouts-main.md

```diff
@@ -213,6 +213,48 @@ There is also some helpful information in the section on
 Some of the information in these references is out of date.
 ```
 
+(detect-candidates-for-summarization)=
+## Detect candidates for summarization
+
+In a large codebase it can be hard to find all the functions that may be difficult for the Prover.
+A traditional approach would be to run a simple parametric rule to explore all functions in the 
+relevant contracts, and studying resulting potential timeouts. 
+However such an approach prolongs the feedback loop of working with the Prover.
+
+As an alternative approach, the Prover supports an {ref}`overapproximating <glossary>` _auto-summarization_ mode.
```

Comment:
I always use "{term}`overapproximating`", tioli of course

Reply:
didn't know about it!